holoviews/plotting/bokeh/raster.py

```diff
@@ -181,6 +186,14 @@ def get_data(self, element, ranges, style):
         img = np.dstack([element.dimension_values(d, flat=False)
                          for d in element.vdims])
 
+        if self.clim_percentile:
+            if isinstance(self.clim_percentile, (int, float)):
+                low, high = np.percentile(img, (self.clim_percentile, 100 - self.clim_percentile))
+            else:  # True
+                low, high = np.percentile(img, (2, 98))
```

Comment:
This will never run, because bool will go into the first statement

Reply:
Essential making it `np.percentile(img, (1, 99))`

Reply:
Wow never knew about this, but now it works!

<img width="1102" alt="image" src="https://github.com/holoviz/holoviews/assets/15331990/1834bc24-2c04-4f8b-a165-def018717a70">